src/api/data_processing.py

```diff
@@ -0,0 +1,593 @@
+import sys, os, logging, datetime, copy
+import pandas as pd
+import numpy as np
+import matplotlib
+import matplotlib.pyplot as plt
+
+sys.path.append("..")
+from epreader import *
+from datetimeep import *
+from typing import Union
+
+
+class DataProcessing:
+    def __init__(
+        self,
+        data_path: str = None,
+        data_source: str = None,
+        timestamp_column_name: str = None,
+    ):
+        """Instantiate a data processing object to load datasets and manipulate data before feeding it to the verification process.
+
+        Args:
+            data (str): Path to the data (CSV format) to be loaded for processing.
+            data_source (str): Data source name. Use `EnergyPlus` or `Other`.
+            timestamp_column_name (str): Name of the column header that contains the time series timestamps.
+        """
+
+        self.data = None
+
+        if data_path == None:
```

Comment:
@leijerry888 - I actually have one comment. It looks like we currently have inconsistent if statement comparing a variable value to `None`. Should we convert them all to `is None`? [Here](https://stackoverflow.com/questions/3257919/what-is-the-difference-between-is-none-and-none)'s a reference.

Reply:
@lymereJ Good point. Done.